utils/bccmd/Makefile

```diff
@@ -0,0 +1,45 @@
+# SPDX-Identifier-License: GPL-2.0-or-later
+#
+# Copyright (C) 2023 Vincent Tremblay <vincent@vtremblay.dev>
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bccmd
+PKG_VERSION:=6.0
+PKG_RELEASE:=1
+
+PKG_MAINTAINER=Vincent Tremblay <vincent@vtremblay.dev>
+
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE_FILES:=COPYING
+
+PKG_FIXUP:=autoreconf
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/bccmd/${PKG_NAME}/tar.gz/v$(PKG_VERSION)?
```

Comment:
Please use regular git, look at PKG_SOURCE_PROTO for examples,

Reply:
Thanks for the hint. I have updated the file and re-tested it.

Reply:
> Please use regular git, look at PKG_SOURCE_PROTO for examples,

hmmm, may I ask if there's any problem with codeload?